lib/delta_filters.rb

```diff
@@ -0,0 +1,23 @@
+module Inbox
+  class DeltaFilters
+
+    attr_reader :filter_map
+
+    def self.build_exclude_types(object_types)
+      new.build_exclude_types(object_types)
+    end
+
+    def initialize(filter_map: API::OBJECTS_TABLE)
```

Comment:
WOMP. :frowning: I know it's a reality, but supporting EOL-ed versions of Ruby is no fun. I'll make this change.